uid2_client/input_util.py

```diff
@@ -119,3 +120,11 @@ def normalize_and_hash_phone(phone):
     if not is_phone_number_normalized(phone):
         raise ValueError("phone number is not normalized: " + phone)
     return get_base64_encoded_hash(phone)
+
+
+def get_datetime_iso_format(timestamp):
```

Comment:
Nitpick, but I'm not sure about having this as a util function. Would probably just call `since_timestamp.isformat()` in `get_identity_buckets` and add a unit test that  `since_timestamp` is in the correct format in the request body.

Reply:
I removed this function and called the `since_timestamp.isoformat`() in `get_identity_buckets`